### PR TITLE
Add cipher list for OpenSSL

### PR DESCRIPTION
**Pull request by Udera**
Fixes https://github.com/vexim/vexim2/issues/176

### CODE REVIEW

File: docs/configure [hunk 1]
```diff
@@ -310,10 +310,17 @@ tls_advertise_hosts = *
 tls_certificate = /etc/exim4/exim.crt
 tls_privatekey = /etc/exim4/exim.key
 tls_dhparam = /etc/exim4/dhparam.pem
+
+# A list of secure ciphers for GnuTLS which is used in Debian/Ubuntu
 tls_require_ciphers = ${if =={$received_port}{25}\
     {NORMAL:%COMPAT:-ARCFOUR-128:-ARCFOUR-40:-MD5:-VERS-SSL3.0}\
     {SECURE128:+AES-256-CBC:+CAMELLIA-256-CBC:-ARCFOUR-128:-ARCFOUR-40:-MD5:-VERS-SSL3.0}}
 
+# If OpenSSL is used (CentOS, FreeBSD), comment the previous line (tls_require_ciphers)
```
Comment (rimas-kudelis) [inline docs/configure]: It's a block, not a line.

Comment (rimas-kudelis) [inline docs/configure]: Or 'lines' at least. :)

Comment (Udera) [inline docs/configure]: True, I updated the PR.

Comment (rimas-kudelis) [inline docs/configure]: No you haven't?

Comment (rimas-kudelis) [inline docs/configure]: Not here, I mean. :)

Comment (Udera) [inline docs/configure]: Now, it's fixed in both places :D

**Comment by rimas-kudelis** [comment]
great! :D

**Pull request merged**